lib/9pfs/README.md

```diff
@@ -0,0 +1,166 @@
+# 9pfs: Unikraft's 9p Virtual Filesystem
+
+With `9pfs`, you can create virtual filesystem devices (`uk_9pdev` in the context of Unikraft) and make them directly accessible between a guest OS and a host as a pass-through filesystem.
+This filesystem uses the [9P network protocol](https://en.wikipedia.org/wiki/9P_(protocol)) for communication between the two instances.
+
+This protocol has a few operations that are all initiated by the clients.
+Each request is satisfied by a single associated response from the server.
+
+## Key Data Structures
+
+### Mount Data Structure
+
+```c
+struct uk_9pfs_mount_data {
```

Comment:
Align the structure members like they are done in the `uk_9pfs_file_data` and `uk_9pfs_node_data` below.